internal/scripts/scripts.go

```diff
@@ -71,6 +73,8 @@ func (c *ContentValue) Attr(name string) (Value, error) {
 		return starlark.NewBuiltin("Content.write", c.Write), nil
 	case "list":
 		return starlark.NewBuiltin("Content.list", c.List), nil
+	case "symlink":
+		return starlark.NewBuiltin("Content.symlink", c.Symlink), nil
```

Comment:
This PR has two unrelated changes apparently, or is the new method connected to implicit directories (not sure about what that means yet)?

Reply:
This PR is now only about `content.symlink`. The other changes were removed.